### DISCUSSION TITLE
Weird webp loading bug?

### DISCUSSION THREAD

**Discussion by FediVideos** [mastodon/mastodon#23264] [General]
Is anyone having problems loading webp images?

There's an image here which won't load on its original page:

https://mastodon.green/@pjakobs/109749588056832226

However, if I load that post within my own server it loads fine.

Anyone know what might be going on?

**Comment by pljakobs** [comment]
since I am the one quoted as the example, I can confirm that the img object is a 404 on my mastodon instance (mastodon.green). 
```<img src="https://mastodon.green/media_proxy/109749413715221128/small" alt="Symbolbild (künstlerische Ansicht) des Verkehrs der Zukunft. Bild: DLR" title="Symbolbild (künstlerische Ansicht) des Verkehrs der Zukunft. Bild: DLR" style="object-position: 50% 50%;">```

on the original instance, this image looks like this:

```<img src="https://helmholtz.social/system/media_attachments/files/109/749/411/909/227/632/small/76bf2357a0bc404c.webp" srcset="https://helmholtz.social/system/media_attachments/files/109/749/411/909/227/632/original/76bf2357a0bc404c.webp 1149w, https://helmholtz.social/system/media_attachments/files/109/749/411/909/227/632/small/76bf2357a0bc404c.webp 640w" sizes="548px" alt="Symbolbild (künstlerische Ansicht) des Verkehrs der Zukunft. Bild: DLR" title="Symbolbild (künstlerische Ansicht) des Verkehrs der Zukunft. Bild: DLR" style="object-position: 50% 50%;">```

not sure if that's helpful.

pj

**Comment by pljakobs** [comment]
Another post that does not allow me to view the embedded picture
https://chaos.social/@lavievagabonde/109765888950834508
